send/generic_send

```diff
@@ -168,6 +174,19 @@ ERR_CODE=$?
 if [ $ERR_CODE -eq 124 ]; then
 	echo "Communication with slave script was timed out with return code: $ERR_CODE (Warning: this error can mask original error 124 from peer!)" >&2
 else
+	if [ "$DESTINATION_TYPE" = "$DESTINATION_TYPE_URL" ]; then
+		#the return code from curl is saved in STDOUT
+		if [ 200 -ne "$STDOUT" ]; then
+			ERR_CODE=$STDOUT
+			cat "$TMPFILE" >&2
+		else
+			ERR_CODE=0
+			cat "$TMPFILE"
+		fi
+	else
+		#if this is not url destination, than we need to print stdout from slave script
+		echo "$STDOUT"
```

Comment:
I think that after the changes which were made in this pull request, this echo will print http code instead of stdout from the slave script (which is in tmpfile now). So, as a result, the slave script message will not be printed at all if the destination type is not URL.